contracts/serverless-v2/Jobs.sol

```diff
@@ -118,6 +134,7 @@ contract Jobs is
         uint256 execStartTime;
         address jobOwner;
         address gateway;
+        uint256 firstOutputExecutionTime;   // in milliseconds
```

Comment:
you can just use `executionTime`. Along with a comment that it is always be exec time of first output submitted.